jwala-webservices/src/main/java/com/cerner/jwala/ws/rest/v1/service/webserver/WebServerServiceRest.java

```diff
@@ -3,69 +3,114 @@
 import com.cerner.jwala.common.domain.model.group.Group;
 import com.cerner.jwala.common.domain.model.id.Identifier;
 import com.cerner.jwala.common.domain.model.webserver.WebServer;
+import com.cerner.jwala.common.exec.CommandOutput;
 import com.cerner.jwala.ws.rest.v1.provider.AuthenticatedUser;
 import com.cerner.jwala.ws.rest.v1.service.webserver.impl.JsonControlWebServer;
 import com.cerner.jwala.ws.rest.v1.service.webserver.impl.JsonCreateWebServer;
 import com.cerner.jwala.ws.rest.v1.service.webserver.impl.JsonUpdateWebServer;
+import io.swagger.annotations.*;
 
 import javax.ws.rs.*;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 
+@Api(value = "/webservers", tags = "webservers")
 @Path("/webservers")
 @Produces(MediaType.APPLICATION_JSON)
 public interface WebServerServiceRest {
 
     @GET
-    Response getWebServers(@QueryParam("groupId") final Identifier<Group> aGroupId);
+    @Consumes(MediaType.MULTIPART_FORM_DATA)
+    @ApiOperation(value = "Get all the web servers in a group",
+            response = WebServer.class
+    )
+    Response getWebServers(@ApiParam(value = "The web servers' group ID", required = true) @QueryParam("groupId") final Identifier<Group> aGroupId);
 
     @GET
     @Path("/{webserverId}")
-    Response getWebServer(@PathParam("webserverId") final Identifier<WebServer> aWebServerId);
+    @ApiOperation(value = "Get a single web server by ID",
+            response = WebServer.class
+    )
+    Response getWebServer(@ApiParam(value = "The web server's ID", required = true) @PathParam("webserverId") final Identifier<WebServer> aWebServerId);
 
     @POST
     @Consumes(MediaType.APPLICATION_JSON)
-    Response createWebServer(final JsonCreateWebServer aWebServerToCreate,
+    @ApiOperation(value = "Create a web server",
+            response = WebServer.class
```

Comment:
Same here, add a note stating it creates a webserver in Jwala DB, actual webserver on the server is created upon generation.